lib/seed/config.rb

```diff
@@ -4,6 +4,8 @@ class Config
     def initialize
       @type = ENV["SEED_TYPE"] ||
         case SimpleServer.env
+        when "android_review"
```

Comment:
This is the `SIMPLE_SERVER_ENV` that the mobile test suite will set when it spins up a test server.

Reply:
Will this be set somewhere in `app.json`?

Reply:
It's being set manually with a `heroku config:set` CLI command in the Github action steps. I toyed with an idea to introduce a `.env.android-review` dotenv file, but didn't have luck with it yet.

Reply:
And `heroku config:set` overrides `app.json` I am guessing.

Reply:
Actually, `app.json` is only used for _review apps_, which spawn off of server PRs. The android repo is going to be creating its own heroku apps per _android_ PR without using review apps, so the `app.json` is completely ignored.